util/chplenv/third_party_utils.py

```diff
@@ -57,6 +58,41 @@ def handle_la(la_path):
                             args.append(tok)
     return args
 
+#
+# Return linker arguments required to link with a library
+# known to pkgconfig. The pkg can be a path to a .pc file or
+# the name of a system-installed package or the name of
+# a third-party package.
+#
+# if system=True, searches for a system-instaled package.
+# if static=True, uses --static (suitable for static linking)
+@memoize
+def pkgconfig_get_link_args(pkg, ucp='', system=True, static=True):
+  havePcFile = pkg.endswith('.pc')
+  pcArg = pkg
+  if not havePcFile:
+    if system:
+      # check that pkg-config knows about the package in question
+      subprocess.check_call(['pkg-config', '--exists', pkg])
```

Comment:
Because check_call is more Google-able. I could use run_command. Would that be better?